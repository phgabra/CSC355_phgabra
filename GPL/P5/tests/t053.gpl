double i = 0.0000001 && 0.000001;
double j = (0.0000001 && 0.000001) && (0.0000001 && 0.0);
double k = (0.0000001 && 0.000001) || (0.0000001 && 0.0);
double l = (0.0000001 && 0.000001) && (0.0000001 || 0.0);
