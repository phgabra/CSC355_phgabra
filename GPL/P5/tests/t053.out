gpl.cpp::main()
  input file(tests/t053.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double i = 1
double j = 0
double k = 1
double l = 1
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
